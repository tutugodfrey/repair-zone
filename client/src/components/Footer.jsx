import React, { Component } from 'react';
import Div from './elementComponents/Div.jsx';
import Link from './elementComponents/Link.jsx';

export default class Footer extends Component {
  footerContent() {
    return (
      <div id="footer-div" className="row align-items-center bg-success" >
        <Div
        divClass="col-6 offset-5 col-md-2 offset-md-3 pl-0 pr-0"
        id=""
        content={ <Link href="#" linkClass="text-white" linkText="Contact us" />}
        />
        <Div
        divClass="col-6 offset-5 col-md-2 offset-md-0 pl-0 pr-0"
        id=""
        content={<Link href="#" linkClass="text-white" linkText="About us" />}
        />
        <Div
        divClass="col-6 offset-5 col-md-2 offset-md-0 pl-0 pr-0"
        id=""
        content={<Link href="#" linkClass="text-white" linkText="Term of Use" />}
        />
      </div>
    );
  }

  render() {
    return this.footerContent();
  }
}