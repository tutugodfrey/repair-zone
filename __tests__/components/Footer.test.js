import React from 'react';
import { shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import Footer from '../../client/src/components/Footer.jsx';

describe('<Footer />', () => {
  const wrapper = shallow(
    <Footer />
  );
  test('should match the snapshot of the component', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
