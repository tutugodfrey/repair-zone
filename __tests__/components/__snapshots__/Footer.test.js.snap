// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should match the snapshot of the component 1`] = `
<div
  className="row align-items-center bg-success"
  id="footer-div"
>
  <Div
    content={
      <Link
        href="#"
        linkClass="text-white"
        linkText="Contact us"
      />
    }
    divClass="col-6 offset-5 col-md-2 offset-md-3 pl-0 pr-0"
    id=""
  />
  <Div
    content={
      <Link
        href="#"
        linkClass="text-white"
        linkText="About us"
      />
    }
    divClass="col-6 offset-5 col-md-2 offset-md-0 pl-0 pr-0"
    id=""
  />
  <Div
    content={
      <Link
        href="#"
        linkClass="text-white"
        linkText="Term of Use"
      />
    }
    divClass="col-6 offset-5 col-md-2 offset-md-0 pl-0 pr-0"
    id=""
  />
</div>
`;
